### PR TITLE
Allow for simpler fake plate definition (docs)

### PR DESCRIPTION
**Pull request by ximenesuk**
This docs PR supports gh-2182

### CODE REVIEW

File: docs/sphinx/developers/generating-test-images.txt [hunk 1]
```diff
@@ -42,17 +42,18 @@ To generate a simple plate file:
...
+    touch "default-screen&screens=1.fake"
 
-This will create a single plate which is, by default, within a containing
-screen. To create a plate with no containing screen add ``&screens=0`` to the
-key-value pairs in the file name. As above a :file:`.fake.ini` file can be
-used.
+These will each create a single plate which is within a containing screen,
+by default in the first two cases. To create a plate with no containing screen
+add ``&screens=0`` to the key-value pairs in the file name. As above a
+:file:`.fake.ini` file can be used.
```
Comment (sbesson) [inline docs/sphinx/developers/generating-test-images.txt]: I am confused  by this paragraph. With 722701bd6d088f2537e1a10867583b33e705edce, the first two example are generating an orphaned plate. Isn't it the opposite, i.e. no screen by default and to add to a screen, `&screens=1` should be appended to the file name?

Comment (ximenesuk) [inline docs/sphinx/developers/generating-test-images.txt]: Sorry, I'll double check the behaviour and try to make the paragraph clearer.

Comment (ximenesuk) [inline docs/sphinx/developers/generating-test-images.txt]: Fixed below. I blame my extremely badly worded commit: https://github.com/ximenesuk/bioformats/commit/722701bd6d088f2537e1a10867583b33e705edce which states the opposite of what it does!

**Comment by ximenesuk** [comment]
There are still some rows in the key-value table that are unexplained and, for me, not obvious from the code.

 + ` thumbnail` : is this just whether there is a thumbnail in the file itself?
 + `scaleFactor`: what is scaled here?
 + `color` : what does this color apply to?
 +  `color_x` : this is dependent on the one above.

**Comment by sbesson** [comment]
@ximenesuk: as far as I can tell from the source code:
- `color_x` sets the color of the `x` channel (expressed in rgba)
- `color` sets the default color for all channels (and is overriden by `color_x`)
- `scaleFactor` is a scaling factor for the pixel values on each plane
- `thumbnail` toggles on/off the [CoreMetadata.thumbnail](http://downloads.openmicroscopy.org/bio-formats/5.1.7/api/loci/formats/CoreMetadata.html#thumbnail). I assume this would be used in the case of fake multi-series images @melissalinkert ?

**Comment by melissalinkert** [comment]
```thumbnail``` would only be used if multiple series are present; by default it is always false.  Setting it to true for a specific series means that the series is a lower resolution/thumbnail/icon representing one of the other defined series.  It's not a common use case for .fake data - if I recall, we only added it for completeness.

**Comment by ximenesuk** [comment]
Thanks both. For `thumbnail` I will simply add "toggles on/off the `CoreMetadata.thumbnail`". If anyone reall wants to use it I guess they will know what that means.

**Comment by ximenesuk** [comment]
@sbesson I'll re-list this for Monday unless you get time to check a local docs build.

**Comment by sbesson** [comment]
@ximenesuk: quickly browsing through the text, everything reads fine, thanks. Please list for Monday for a final review.

**Comment by sbesson** [comment]
Staged at http://www.openmicroscopy.org/site/support/bio-formats5.2-staging/developers/generating-test-images.html. Reads well. Thanks @ximenesuk.

**Pull request merged**